### PR TITLE
MODINVOICE-41-GET /voucher/vouchers/<id>

### PR DESCRIPTION
**Pull request by KVupp**
## Purpose
https://issues.folio.org/browse/MODINVOICE-41

## Approach
Adding basic implementation, add unit tests

## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [x] Were permissions changed, added, or removed?
  - [x] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -92,6 +92,14 @@
           "methods": ["GET"],
           "pathPattern": "/invoice/invoice-number",
           "permissionsRequired": ["invoice.invoice-number.item.get"]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/voucher/vouchers/{id}",
```
Comment (pautau-s) [inline descriptors/ModuleDescriptor-template.json]: Maybe it is worth placing everything related to vouchers in a separate interface.

File: ramls/voucher.raml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+#%RAML 1.0
+
+title: Voucher
+version: v0.1
```
Comment (piotr-kalashuk) [inline ramls/voucher.raml]: Probably better to keep in sync with interface version i.e. `1.0`

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -92,6 +92,14 @@
           "methods": ["GET"],
           "pathPattern": "/invoice/invoice-number",
           "permissionsRequired": ["invoice.invoice-number.item.get"]
+        },
+        {
+          "methods": ["GET"],
```
[the remark this answers is not in the document]
Comment (piotr-kalashuk) [reply descriptors/ModuleDescriptor-template.json]: Note: according to [naming conventions](https://dev.folio.org/guidelines/naming-conventions/#interfaces) the name should be `voucher`

File: descriptors/ModuleDescriptor-template.json (hunk 1 above)
Comment (KVupp) [reply descriptors/ModuleDescriptor-template.json]: Done

File: descriptors/ModuleDescriptor-template.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (KVupp) [reply descriptors/ModuleDescriptor-template.json]: Changed

File: ramls/voucher.raml (hunk 2 above)
Comment (KVupp) [reply ramls/voucher.raml]: changed this raml as well as the invoice raml to be in sync with interface version

**Pull request merged**